src/components/CookieBanner/CookieBanner.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import { CgClose } from "react-icons/cg";
+import CookieIcon from '../../img/cookieIcon.png'
+import VhiLogo from '../../img/cookieVhiLogo.png'
+
+export default function CookieBanner() {
+
+    const [isVisible, setIsVisible] = useState(false);
+
+    function ShowCookieBanner() {
+        setIsVisible(true);
+    }
+
+    function hideCookieBanner() {
```

Comment:
the name should start with "on" , hence, "onClickHide"

Reply:
I've resolved this.